### PR TITLE
{tools}[GCCcore/10.2.0] numdiff v5.9.0

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/263727219e235d61e3de004af92de6d3 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/2e22ac20e5c67a8d7c846a266bd2f012 for a full test report.

File: easybuild/easyconfigs/n/numdiff/numdiff-5.9.0-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'numdiff'
+version = '5.9.0'
+
+homepage = 'https://savannah.nongnu.org/projects/numdiff'
```
Comment (bedroge) [inline easybuild/easyconfigs/n/numdiff/numdiff-5.9.0-GCCcore-10.2.0.eb:6]: Just a very minor thing, but it looks like the "official" homepage of this project is https://www.nongnu.org/numdiff/ (the Savannah page also has a "homepage" link that points to this page)?

```suggestion
homepage = 'https://www.nongnu.org/numdiff/'
```

Comment (branfosj) [reply easybuild/easyconfigs/n/numdiff/numdiff-5.9.0-GCCcore-10.2.0.eb]: Changed.

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 20.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.8.10
See https://gist.github.com/8f99ffdf1a492492f58d92abf34aa46f for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
pg-interactive.hpc.rug.nl - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/ba7c9f477720728d52f4ecca39bb6ec0 for a full test report.

**Review by bedroge — APPROVED** [review]
lgtm

**Comment by bedroge** [comment]
Going in, thanks @branfosj!

**Pull request merged**